js/src/edit.js

```diff
@@ -0,0 +1,111 @@
+/* global wpseoPostScraperL10n */
+
+import { createStore, applyMiddleware, combineReducers } from "redux";
+import thunk from "redux-thunk";
+import logger from "redux-logger";
+import React from "react";
+import ReactDOM from "react-dom";
+import { Provider } from "react-redux";
+import { IntlProvider, addLocaleData } from "react-intl";
+
+import analysis from "yoast-components/composites/Plugin/ContentAnalysis/reducers/contentAnalysisReducer";
+import activeKeyword from "./redux/reducers/activeKeyword";
+import ContentAnalysis from "./components/contentAnalysis/ReadabilityAnalysis";
+import SeoAnalysis from "./components/contentAnalysis/SeoAnalysis";
+
+/**
+ * This should be the entry point for all the edit screens. Because of backwards compatibility we can't change this at once.
+ */
+
```

Comment:
double asterisks are meant for docblocks, multiline comments should have just one asterisk, but this is a single line comment. Either should start with `//` or be split in two lines and use a single asterisk.